sdks/python/apache_beam/dataframe/expressions.py

```diff
@@ -45,6 +45,45 @@ def lookup(self, expr):  #  type: (Expression) -> Any
     return self._bindings[expr]
 
 
+class PartitioningSession(Session):
+  def evaluate(self, expr):
```

Comment:
```suggestion
class PartitioningSession(Session):
  """An extension of Session that ensures input dataframes are split into at least `len(df)` separate partitions and input in a random order. For testing only.
  """
  def evaluate(self, expr):
```

Or something like that :)